kylin-it/src/test/java/org/apache/kylin/provision/BuildCubeWithEngine.java

```diff
@@ -142,6 +142,8 @@ public static void beforeClass(String confDir) throws Exception {
         System.setProperty(KylinConfig.KYLIN_CONF, confDir);
         System.setProperty("SPARK_HOME", "/usr/local/spark"); // need manually create and put spark to this folder on Jenkins
         System.setProperty("kylin.hadoop.conf.dir", confDir);
+        System.setProperty("kylin.source.hive.warehouse-dir", "/apps/hive/warehouse");
```

Comment:
Set this property in `kylin-it/pom.xml` maybe better and easy to find by other user.